docs/Example HOWTO.md

```diff
@@ -320,3 +383,150 @@ dns2.example.com.    IN      A       192.168.0.20
 rds1.example.com.    IN      A       192.168.0.50
 rds2.example.com.    IN      A       192.168.0.51
 ```
+### Checking the `DNS` records for Multiple RDSs
+
+
+```
+$dig _nmos-register._tcp.example.com PTR
+ <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.amzn2.5.2 <<>> _nmos-register._tcp.example.com PTR
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 48397
+;; flags: qr aa rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 2, ADDITIONAL: 3
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;_nmos-register._tcp.example.com. IN	PTR
+
+;; ANSWER SECTION:
+_nmos-register._tcp.example.com. 3600 IN PTR	reg-api-2._nmos-register._tcp.example.com.
+_nmos-register._tcp.example.com. 3600 IN PTR	reg-api-1._nmos-register._tcp.example.com.
+
+;; AUTHORITY SECTION:
+example.com.		3600	IN	NS	dns2.example.com.
+example.com.		3600	IN	NS	dns1.example.com.
+
+;; ADDITIONAL SECTION:
+dns1.example.com.	3600	IN	A	192.168.0.18
+dns2.example.com.	3600	IN	A	192.168.0.20
+
+;; Query time: 0 msec
+;; SERVER: 10.0.50.59#53(10.0.50.59)
+;; WHEN: Tue Sep 06 07:29:22 UTC 2022
+;; MSG SIZE  rcvd: 178
+
+
+```
+
+As can be seen the DNS server provides two PTRs to the query.  Details on prioritization and TXT can be checked as well:
```

Comment:
The following snippet doesn't seem to include checking the `TXT` records?

Reply:
Will rerun this section as overall verification of the final DNS BIND9 database file below.

Reply:
hi @garethsb  - I reran the test sections with the full BIND9 db and found one problem that I corrected in the merge request.  If you can take a quick look at those two then I think we are ready for the merge?  (Needed the . at the end of the NS resource record for example.com.)

Reply:
Great! Should we add a `dig` for the `TXT` records as well as the `SRV` though?